tests/util/images.go

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+
+package util
+
+import (
+	"fmt"
+
+	"kubevirt.io/kubevirt/tests/flags"
+)
+
+func GetUtilityImageFromRegistry(imageName string) string {
```

Comment:
I would prefer not to see additions under any generic `util` package.

Can this be placed in a more focused package?
E.g. `libimageregistry` or `libregistry`.

Reply:
@EdDev, what do you think about adding the function to `libinfra` instead? Does it look like a suitable place? I do not have a strong opinion on that, tbh, so I am also fine with creating a new package.

Reply:
If I only knew what this `libinfra` represents.
Based on git history, it got helpers from a prev test file names `infra_test.go`.

Do you think the image registry is related somehow? I just do not know.

Reply:
@dhiller , can you please suggest what this `libinfra` represents exactly?

Reply:
My guess was that it stand for `infrastructure` stuff. Currently, it includes smth related to certificates, http, node-labeler, etc. Just assumed that utility container images might fall into the infra category as well.

Reply:
Well, it is better than `utils`.
It does have the potential to collect another version of unrelated stuff.

I'll leave the decision for you, I have no strong feelings about it.

Reply:
Tried to move the function to `libinfra`. After importing it where needed, got a circular dependency:
```
ERROR: /root/go/src/kubevirt.io/kubevirt/tests/testsuite/BUILD.bazel:3:11: in go_library rule //tests/testsuite:go_default_library: cycle in dependency graph:
    //cmd/dump:dump (3e1517af6a4f7eb088e2179e4d66b4fd00494be72ad1028a2ff1c0eb967eafe2)
    //cmd/dump:go_default_library (3e1517af6a4f7eb088e2179e4d66b4fd00494be72ad1028a2ff1c0eb967eafe2)
    //tests/reporter:go_default_library (3e1517af6a4f7eb088e2179e4d66b4fd00494be72ad1028a2ff1c0eb967eafe2)
.-> //tests/testsuite:go_default_library (3e1517af6a4f7eb088e2179e4d66b4fd00494be72ad1028a2ff1c0eb967eafe2)
|   //tests/libstorage:go_default_library (3e1517af6a4f7eb088e2179e4d66b4fd00494be72ad1028a2ff1c0eb967eafe2)
|   //tests/libinfra:go_default_library (3e1517af6a4f7eb088e2179e4d66b4fd00494be72ad1028a2ff1c0eb967eafe2)
|   //tests:go_default_library (3e1517af6a4f7eb088e2179e4d66b4fd00494be72ad1028a2ff1c0eb967eafe2)
|   //tests/libwait:go_default_library (3e1517af6a4f7eb088e2179e4d66b4fd00494be72ad1028a2ff1c0eb967eafe2)
`-- //tests/testsuite:go_default_library (3e1517af6a4f7eb088e2179e4d66b4fd00494be72ad1028a2ff1c0eb967eafe2)
ERROR: Analysis of target '//cmd/dump:dump' failed; build aborted
```
Seems that there might be some mess to take care of before doing that.

Reply:
> Looks like I am not alone with that pain
> 
> https://github.com/kubevirt/kubevirt/blob/9db1b1301b2dbd8261020006b67de1a2663f6cbc/tests/migration/eviction_strategy.go#L257-L261

Here I am :slightly_smiling_face: :raising_hand_man:

Reply:
@vasiliy-ul , can you try and place it in the previously suggested new package?

`libimageregistry` or `libregistry` ?

Reply:
Well, I tried to fix the cycle. But eventually faced a few more. There is a need to decouple a lot of stuff to fully address the problem, and the refactoring turns out pretty messy at the end. So yeah, probably for now if that is okay, I will go with a new `libregistry` package.

I will take one more look and probably open a follow-up PR with some attempts to move things around in the hope to make things a bit better.